### ISSUE TITLE
Cinnamon: GWL - Stuck file opperation progress state, thin line remains

### ISSUE THREAD

**Issue by DaveBlack** [linuxmint/mint-19.1-beta#78]
**Describe the bug**
GWL seems to have inherited a small ITM issue. Sometimes after file operations have completed, the completed progress state remains stuck, clicking apps icon causes a thin line to  remain on the left.

**To Reproduce**
1. Copy files with Nemo
2. (if remember correctly) Once seen with cancelled, almost completed, Firefox download

**Expected behavior**
App icon should refresh upon file operation completion

**Frequency**
Does the issue happen:
- [ ] Always
- [x] Randomly

**Edition (Desktop environment):**
In which edition is this happening?
- [x] Cinnamon Edition
- [ ] MATE Edition
- [ ] Xfce Edition

**Live or post-install:**
Indicate if this is happening after the installation, or during the live session:
- [x] Post-install
- [ ] Live session

**Screenshots**
![line remains after stuck progress state](https://user-images.githubusercontent.com/13056699/50045778-ee036000-0090-11e9-8298-e6a1da6d85d8.png)

**Locale:**
LANG=en_GB.UTF-8

**Drivers:**
NVIDIA 390.77

**Comment by jaszhix** [comment]
Can you attach your settings in ~/.cinnamon/configs/grouped-window-list@cinnamon.org? Not able to reproduce this after a few attempts.

**Comment by DaveBlack** [comment]
Only one setting changed...
Middle click action - Launch new app instance

Everything else is default.

Still trying to find an exact way to reproduce this issue. Have seen it on 5 very different systems (base machine installs) with ITM, and 3 with GWL (2 base machine installs, 1 VirtualBox).

**Comment by DaveBlack** [comment]
Still requires further testing, but now wondering if this issue is more likely to occur when all main Nemo windows are closed, except for smaller file transfer progress window.

**Comment by DaveBlack** [comment]
In case helpful, just found images grabbed a week ago.

After Nemo file copy (stuck progressOverlay?), all windows closed
![screenshot from 2018-12-10 22-09-17](https://user-images.githubusercontent.com/13056699/50171472-74ff4500-02ea-11e9-866c-0ab2e5572eb5.jpg)

Opening 'Files' Nemo window shows thin line on left
![screenshot from 2018-12-10 22-10-02](https://user-images.githubusercontent.com/13056699/50171533-9b24e500-02ea-11e9-95ea-09f5572451f2.jpg)

Closing Nemo window, thin line remains
![screenshot from 2018-12-10 22-10-11](https://user-images.githubusercontent.com/13056699/50171591-be4f9480-02ea-11e9-8799-59214547a61a.jpg)

While above images were from a virtual machine with 3D acceleration disabled (to prevent a vbox Cinnamon display 'bloom' issue), have seen the same stuck progress and thin line remaining on host with NVIDIA GPU.

**Comment by DaveBlack** [comment]
Based on my above theory of closing main Nemo windows and leaving the small file copy progress window open, just been able to cause the issue of the stuck progress indication to happen twice. This is on a very different system with a quad core Atom CPU and Intel GPU.

Here's what I did (may, or may not work for you)...

1. Open two Nemo windows side by side (different drives/partitions)
2. Drag large file from one window to the other
3. Close both Nemo windows, leaving just the queued file copy progress window (sorry don't know its name)
4. After file copy completes, queued file copy progress window closes, but Nemo ('Files') GWL progress notification appears to be stuck
5. Clicking 'Files' launches Nemo, stuck progress notification now changes to a thin line
6. Closing Nemo window leaves the thin line which remains until Cinnamon is restarted, or logout/system restart
